plotly/tools.py

```diff
@@ -2275,6 +2277,25 @@ def create_candlestick(open, high, low, close,
         return dict(data=data, layout=layout)
 
 
+    @staticmethod
+    def create_dendrogram(X, orientation="bottom", labels=None,
+                          colorscale=None, **kwargs):
+        """
```

Comment:
It might be good to keep the docstrings similar across the FigureFactory methods. I used this guideline https://github.com/plotly/dev-docs/blob/master/reference/styleguides/python.md#function-docstrings for the others.